compiler/x/codegen/OpBinary.cpp

```diff
@@ -911,3 +911,904 @@ const TR_X86OpCode::TR_OpCodeBinaryEntry TR_X86OpCode::_binaryEncodings[IA32NumO
       {0x00, 0x00, 0x00, 0},    // AdjustFramePtr
       {0x00, 0x00, 0x00, 0}     // ReturnMarker
    };
+
+#include "codegen/FrontEnd.hpp"
+bool TR_X86OpCode::_UseDeprecatedBinaryEncoding = feGetEnv("TR_UseDeprecatedBinaryEncoding");
+// X87 is only available on X86-32
+#ifdef TR_TARGET_64BIT
+#define X87_________________(x) 0,0,0
+#else
+#define X87_________________(x) (uint8_t)((x & 0xE0) >> 5), (uint8_t)((x & 0x18) >> 3), (uint8_t)(x & 0x07)
+#endif
+
+const TR_X86OpCode::OpCode_t TR_X86OpCode::_binaries[] =
+{
+    { VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_____, 0xcc, 0, ModRM_NONE, Immediate_0 },    // BADIA32Op/int 3
```

Comment:
If you meant to add a line of comment listing the meaning of each column, the name of the macros are designed to explain itself and I believe it is intuitive enough. In addition, it is such a long table, adding a line on the top does not help much as the line would be out of readable panel while looking majority of instructions.

If you meant to comments list values of enums containing VEX_L, PREFIX__, etc.; it is in fact to duplicate the code. It would be a hassle for maintaining. It is expected that anyone who tries to understand, modifies and/or debug this array to first understand the structure TR_X86OpCode::OpCode_t, a simple structure; and once understood, this array, TR_X86OpCode::_binaries[] is very intuitive. It is dangerous trying to interpret content of any array without understanding it element type. In addition, most, if not all, modern IDE can locate TR_X86OpCode::OpCode_t from any entry of this array. Therefore, I personally dislike the idea to comment on any structure details or enum values here, as it would likely induce the reader not to read TR_X86OpCode::OpCode_t before reading this array.